Tests/SonarQube.TeamBuild.PreProcessor.Tests/DataModel/QualityProfile.cs

```diff
@@ -0,0 +1,46 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="QualityProfile.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace SonarQube.TeamBuild.PreProcessor.Tests
+{
+    internal class QualityProfile
```

Comment:
I created a series of data classes to allow mocking of the data structures on the server e.g. Repository, Rule, QualityProfile.